trunk/NDHMS/Routing/module_GW_baseflow.F

```diff
@@ -339,9 +339,11 @@ subroutine simp_gw_buck(ix,jx,ixrt,jxrt,numbasns,gnumbasns,basns_area,basnsInd,g
 
 #ifdef MPP_LAND
     call gw_sum_real(sum_perc8,numbasns,gnumbasns,basnsInd)
+    call gw_sum_real(sum_perc8_surf,numbasns,gnumbasns,basnsInd)
     call gw_sum_real(ct_bas8,numbasns,gnumbasns,basnsInd)
 #endif
    sum_perc = sum_perc8
+   sum_perc_surf = sum_perc8_surf
```

Comment:
why do it in double precision and then throw it away?

Reply:
there's probably `sum`thing hiding in `gw_sum_real`

Reply:
No idea - I just duplicated what was already in the code. Pre-dates me.